Code/Framework/AzNetworking/AzNetworking/Utilities/EncryptionCommon.cpp

```diff
@@ -51,11 +51,18 @@ namespace AzNetworking
     {
 #if AZ_TRAIT_USE_OPENSSL
 
-        const int32_t errorCode = ERR_get_error();
-            const int32_t systemError = GetLastNetworkError();
-
-            switch (errorCode)
-            {
+        #if(OPENSSL_VERSION_PREREQ(3,0))
+            const char *file, *data, *func;
```

Comment:
These may give you both unused variable errors or uninitialized variable errors (warning as errors) in one of the compilers (gcc or clang). Should probably mark [[maybe_used]] and initialize to nullptr for all of them just in case

Reply:
I can just pass in the values I'm using. I think I'm only using `char* func`